index.js

```diff
@@ -35,8 +41,18 @@ module.exports = function (){
     }
 
     this.getWeb3 = async function (contractABI,contractAddress,provider){
```

Comment:
Why have a web three where we set the contract instance?
I thought you tried to to a singelton approach, then you cant have a global contract instance object i believe

Reply:
My thinking was, if we use singleton, there can be only one active contract at a time (as you have mentioned above). Solution to that is, allow multiple instances of web3js-raw in an app (NO singleton). 

2 ways to use the web3js-raw:

1. web3.js instance with a specific contract instance (access via web3js-raw.ContractInstance)
2. When need only the web3.js, use getWeb3Base. With this need only web3js-raw as a dependency in your project.